### PR TITLE
Unify Spire Stream output for prices

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

File: cmd/spire/cmd_stream.go [hunk 1]
```diff
@@ -71,49 +92,107 @@ func NewStreamCmd(c *spire.Config, f *cmd.FilesFlags, l *cmd.LoggerFlags) *cobra
...
 
+func remapMeta(meta transport.Meta, m map[string]any) map[string]any {
+	m["transport"] = meta.Transport
+	m["topic"] = meta.Topic
+	m["message_id"] = meta.MessageID
+	m["peer_id"] = meta.PeerID
+	m["peer_addr"] = meta.PeerAddr
+	m["received_from_peer_id"] = meta.ReceivedFromPeerID
+	m["received_from_peer_addr"] = meta.ReceivedFromPeerAddr
+	if meta.UserAgent != "" {
```
Comment (konstantinzolotarev) [inline cmd/spire/cmd_stream.go:190]: If `meta.UserAgent == ""` `m["user_agent"]` field will be missing in resulting map, is it required behaviour ? 
Or mb better to add `m["user_agent"]` even it's empty (not sure that it's needed) ?

Comment (BrianBelhumeur) [reply cmd/spire/cmd_stream.go:190]: I prefer the latter as I reference `user_agent` to know which version of which software the message came from, and publish that on the dashboard

File: cmd/spire/messages.go [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+	case *messages.MuSigSignature:
+		v = handleMuSigSignatureMessage(msgType)
+	case *messages.MuSigTerminate:
+		v = handleMuSigTerminateMessage(msgType)
+	case *messages.Greet:
+		v = handleGreetMessage(msgType)
+	default:
+		v = streamType{
+			Data: msg.Message,
+		}
```
Comment (jar-o) [inline cmd/spire/messages.go:86]: if you're intending for `type` and `version` to be required,

https://www.notion.so/Semantics-of-Spire-Stream-Messages-644f46b304734534aba571cfd8a4be20?d=c4e147297a364ec8a3b90e3de37e2e82&pvs=4#85b80f37b9c841faaa510976593701bf

doesn't this break the rule?

**Review by jar-o — APPROVED** [review]
this lgtm with one concern, but its def possible I lack some context that matters given I haven't been in osuite's guts for some time :)

**Pull request merged**